### PR TITLE
WIP-CALS1-5027-sort facility search results by facility name

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5027
-Blocked by https://osi-cwds.atlassian.net/browse/CALS-5659

#### Brief feature description
-Sort facility search results.
-Sort order by facility name. 

#### Does this PR block/effect other stories?
- No

#### Do any commands needs to be run in order to run the project with this PR?
- No

#### Code coverage percent diff as a result of this PR?
- WIP

### CODE REVIEW

**Comment by ratneshraval** [comment]
@sunilb0575 the sort field should be passed in instead of hard coding for `name`

File: app/classes/elastic/query_builder.rb [hunk 1]
```diff
@@ -45,20 +45,29 @@ def self.match_boolean(query_array, from_params, size_params)
           should: combined_query_array
         }
       },
+      sort:
+      [
+        {
+          sort_by_params =>
```
Comment (ratneshraval) [inline app/classes/elastic/query_builder.rb]: please use `:` instead of arrow `=>` notation

**Comment by ratneshraval** [comment]
@sunilb0575 if you add a way to only add sort logic to elasticsearch query when a field is specified, then you can go ahead and merge this branch and not specify search field till backend story is done.

File: app/classes/elastic/query_builder.rb (hunk 1 above)
Comment (sunilb0575) [reply app/classes/elastic/query_builder.rb]: As *sort_by_params* is a string, I am getting an syntax error when used *:*

**Pull request closed without merging**